cli/cmd/agent.go

```diff
@@ -158,6 +161,77 @@ To list all active agents in your environment.
 NOTE: New agents could take up to an hour to report back to the platform.`,
 		RunE: installRemoteAgent,
 	}
+
+	agentCTFCmd = &cobra.Command{
+		Use:   "ctf",
```

Comment:
We tend to be very explicit with the purpose of the commands, ctf (capture the flag) means nothing to the end-user, but if we try things like `deploy`, `bulk-install`, `cloud-install`. it might help the user understand what this "namespace" is about. 

Another option here is to drop the entire `ctf` command/namespace/directory and put the commands as:
* `lacework agent aws-deploy ec2ic`
* `lacework agent aws-deploy ssh`
Or:
* `lacework agent aws-install ec2ic`
* `lacework agent aws-install ssh`

In short, this definitely needs to change, I will think more about which name to use here but it might be nice to ask UX for help.

Reply:
Do we have anyone on the UX team that has helped with command naming in the past?

I had proposed `lacework agent install ec2-ssh` and `lacework agent install ec2-instance-connect` in the `ctf` document. I also like the `aws-deploy` command name myself.

Reply:
I went with `lacework agent install ec2-ssh` and `lacework agent install ec2ic` for now, happy to change this! I decided on `ec2ic` over `ec2-instance-connect` for the sake of brevity, but I can see why we'd want to do `ec2-instance-connect` to have style consistent with `ec2-ssh`